lib/appcontroller_client.py

```diff
@@ -370,3 +379,150 @@ def set_deployment_id(self, deployment_id):
     return self.run_with_timeout(10, 'Set deployment ID request timed out.',
       self.DEFAULT_NUM_RETRIES, self.server.set_deployment_id, self.secret,
       deployment_id)
+
+
+  def get_all_stats(self):
+    """ Queries the AppController for all the stats.
+
+    Returns:
+      All of the application's stats from the AppController
```

Comment:
Change this to mention the type of return value (i.e. A dictionary).
Also, add a period at the end of the sentence.